styleguide/_static/issue-templates/security.md

```diff
@@ -0,0 +1,58 @@
+---
```

Comment:
If we have this template, we need to check whether we can have a reduced one for customers. They should not fill out any of the fields that classify the issue.

Reply:
Sure thing we can have a second template, but I don't really get why we should.
If it has a CVE and is already classified or if they classify correctly I don't see an issue.
Also if an issue gets created, it's type and content is not set in stone (we still can adjust it).
Last but not least, once we see a common/specific of problem of this template we can address it more specifically.

Reply:
Big templates scare off a lot of people. They will rather not write a ticket than having to figure out what to put in those fields.

Or -- even worse -- they fill in nonsense or placeholders.